xml/gnome_start.xml

```diff
@@ -349,7 +349,7 @@
     </itemizedlist>
    </sect3>
    <sect3 xml:id="sec-gnome-start-activities-use">
-    <title>Using the <guimenu>Activities</guimenu> overview</title>
+    <title>Using the <guimenu>Activities</guimenu> Overview</title>
```

Comment:
If the name of the UI element was _Activities Overview_, I'd agree, but here (and for the following occurrences) of _Activities Overview_ I'm wondering if this change is correct. 

Why should we spell 'Overview' with a capital 'O' when 

- we use sentence style for headings? 
- the name of this UI element seems to be _Activities_ ? (which is also correctly tagged with `<guimenu/>` in the XML file) - see also the screenshot on page 23 in the diff-PDF with the corrections (Figure 1.4). I rather think it should be `<guimenu>Activities</guimenu> overview` everywhere.

Reply:
Hi all,
here is a sceenshot from the GNOME user help. They write overview with lower case:
![grafik](https://user-images.githubusercontent.com/77108806/168746305-b49080d9-f507-4121-8237-e2ad459c4f35.png)
I can also do the corrections, if by tomorrow is sufficient

Reply:
Hi Tanja and Julia, 
I agree that we should correct 'overview' everywhere. The reason why I kept the capitalized 'Overview' throughout the entire guide was the simple majority of instances where it was historically spelled with the capital 'O'. Also, lots of external resources have it capitalized for some reason :-)